app/pages/project/instances/instance/InstancePage.tsx

```diff
@@ -147,9 +156,16 @@ export function InstancePage() {
             <InstanceStatusBadge status={instance.runState} />
           </PropertiesTable.Row>
           <PropertiesTable.Row label="vpc">
-            <span className="text-secondary">
-              {primaryVpcId ? VpcNameFromId({ value: primaryVpcId }) : <EmptyCell />}
-            </span>
+            {vpc ? (
+              <Link
+                className="link-with-underline group text-sans-semi-md"
+                to={pb.vpc({ project: instanceSelector.project, vpc: vpc.name })}
+              >
+                {vpc.name}
+              </Link>
+            ) : (
+              <EmptyCell />
+            )}
```

Comment:
I considered doing this in #1882 but was too lazy. Glad to have the push to do it the ugly right way.